frontend/uddhar-frontend/src/shared/components/VolunteerTable.jsx

```
@@ -41,13 +45,24 @@ const VolunteerTable = ({
-                  {volunteer.organization}
-                </td>
                 <td className="py-3 px-6 text-left">{volunteer.location}</td>
+                {setLeaderId && (
+                  <td className="py-3 px-6 text-left">
+                    <button
+                      onClick={() => setLeaderId(volunteer.id)}
+                      disabled={leaderId === volunteer.id}
+                      className={`bg-green-800 text-white px-2 py-1 rounded cursor-pointer ${
+                        leaderId === volunteer.id ? "opacity-50" : ""
+                      }`}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)